avaframe/out3Plot/outCom1DFA.py

```diff
@@ -102,6 +102,40 @@ def plotTrackParticle(outDirData, particlesList, trackedPartProp, cfg, dem):
         # ani.save("testTrackAlr1.gif", writer=writer)
 
 
+def plotTrackParticleAcceleration(outDirData,trackedPartProp, cfg, Tsave):
+    """ Plot time series of tracked partcles
+    Parameters
+    ----------
+    outDirData: str
+        path to output directory
+    particlesList: list
+        list or particles dictionaries
+    trackedPartProp: dict
+        dictionary with time series of the wanted properties for tracked
+        particles
+    cfg : dict
+        configuration read from ini file
+    dem: dict
+        dem dictionary with normal information
```

Comment:
exchange for Tsave

Reply:
actually the trackedParticle dict has its own time info